libraries/AP_ExternalAHRS/AP_ExternalAHRS_VectorNav.cpp

```diff
@@ -815,5 +836,13 @@ void AP_ExternalAHRS_VectorNav::send_status_report(GCS_MAVLINK &link) const
                                        mag_var, 0, 0);
 }
 
+// inject data (for RTCMv3)
+void AP_ExternalAHRS_VectorNav::inject_data(const uint8_t *data, uint16_t len)
+{
+    if (setup_complete && uart->txspace() >= len) {
+        uart->write(data, len);
```

Comment:
Would be good to gate this on detected type. We may need to improve our type detection to support these units more fully in any-case. 

Although I guess it should not do any harm to send RTCM to a device that does not support it.

Reply:
that's what I thought - user wouldn't be injecting anyway if no GPS, and I'd expect it to be harmless if they did